platform_versioned_docs/version-23.3.0/enterprise/configuration/configtables/features_env.yml

```diff
@@ -9,6 +9,11 @@
   Description: > 
     Define the Wave containers service endpoint URL. 
   Value:                'Example: `https://wave.seqera.io`'
+- 
+  Environment variable:            '`TOWER_ENABLE_ARM64`'
+  Description: > 
+    Enable support for ARM64 CPU architecture. Required to enable Graviton instance support in AWS Batch compute environments. 
```

Comment:
This this feature 1:1 AWS Batch only? Or just happens that AWS Batch is the only ARM-type env you can activate right now (but expected that we can add other vendors in future)?